docs/related_link.md

```diff
@@ -0,0 +1 @@
+[支持边云协同终身学习特性，KubeEdge 子项目 Sedna 0.3.0 版本发布！](https://juejin.cn/post/6970866022286884878)
```

Comment:
convert to English

Reply:
The title of the referenced link is in Chinese.